### DISCUSSION TITLE
USB Storage Configuration

### DISCUSSION THREAD

**Discussion by cellularmem** [nextcloud/all-in-one#1245] [Questions] [answered]
I have a local instance of nextcloud aio that is accessible and seems to be running properly.  However, despite activating the external storage module in the admin nextcloud account and specifying NEXTCLOUD_MOUNT:/media/name/disk in my docker yml, the local storage option is not available to me at <nextcloudsite>/settings/admin/externalstorages.

The drive appears to be properly permissioned (drwxr-x--- 3 www-data root 4096 Oct 10 12:17 Backup), but is not visible to the nextcloud container when I follow the troubleshooting outlined in https://github.com/nextcloud/all-in-one/discussions/1138.

Any help would be appreciated.

**Comment by szaimen** [comment, marked as answer]
Hi, did you already try to restart the container from the aio interface? And can you post the whole docker-compose.yml file?
